allensdk/brain_observatory/behavior/stimulus_processing.py

```diff
@@ -694,10 +695,10 @@ def compute_trials_id_for_stimulus(
     trials_sorted = trials_table.sort_values("start_time")
     # Create a placeholder for the trials_id.
     trials_ids = pd.Series(
-        data=np.full(len(stim_pres_sorted), -1, dtype=int),
+        data=np.full(len(stim_pres_sorted), -99),
```

Comment:
please create a constant variable with the value -99 and use that everywhere to represent missing